src/apis/member/api.ts

```diff
@@ -0,0 +1,5 @@
+import type { MemberProfileRes } from './types'
+import { http } from '@utils/http'
+
+export const getMemberProfile = async () =>
+  http.get<unknown, MemberProfileRes>('/api/member/access-token/me')
```

Comment:
```suggestion
  http.get<null, MemberProfileRes>('/api/member/access-token/me')
```
req가 필요하지 않아 null이 맞을 것 같습니다!

Reply:
[반영 완료 ! ](https://github.com/price-offer/offer-fe/pull/175/commits/e1b19a97ac215f4be7d4b0ecd8e7b06d0c7aa3ed)